lib/matplotlib/widgets.py

```diff
@@ -309,6 +310,10 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt='%1.2f',
         valstep : float, optional, default: None
             If given, the slider will snap to multiples of `valstep`.
 
+        orientation : str, optional, default: 'horizontal'
```

Comment:
Canonical [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#id5) recommends curly braces if there is a limited list of choices. The first is implicitly the default:

~~~
orientation : {'horizontal', 'vertical'}
    The orientation of the slider.
~~~